### PR TITLE
Add removeEmptyDataLabel method

### LINKED ISSUE #49: Empty Data Label Issue
The empty data label now stays there even when there is data after moving it to be a subview of the `tableView` rather than `footerView`

### PR DESCRIPTION
**Pull request by TetHat**
Fixes #49

### CODE REVIEW

File: Pump-Tracker/InventoryViewController.swift [hunk 1]
```diff
@@ -174,17 +174,17 @@ public class InventoryViewController: UITableViewController {
...
         /// If no inventory entered yet, show a label centered in the view to tell them how to add some inventory
         let emptyTextLabel = UILabel(frame: CGRect.zero)
-        emptyTextLabel.center = tableView.tableFooterView!.center
         emptyTextLabel.numberOfLines = 0
         emptyTextLabel.textColor = .black
         emptyTextLabel.text = (
             "You haven't entered any data yet.\n\nAdd inventory by clicking the + in the"
             + " upper right-hand corner.")
         emptyTextLabel.textAlignment = .center
+        emptyTextLabel.tag = 100
```
Comment (beetdemguise) [inline Pump-Tracker/InventoryViewController.swift]: Can this be a string? If so, I'd prefer a descriptive tag to an integer.

Comment (TetHat) [reply Pump-Tracker/InventoryViewController.swift]: Unfortunately not. I thought the same. But a view tag in swift is defined as an Int.

Comment (beetdemguise) [reply Pump-Tracker/InventoryViewController.swift]: Ugh, well I don't foresee us using tags ad nauseam so I guess I'm fine with this. Though maybe defining some constants for UI tags is something that should be done.

Comment (TetHat) [reply Pump-Tracker/InventoryViewController.swift]: Something we want to define in Utils or create a new Constants.swift?

Comment (beetdemguise) [reply Pump-Tracker/InventoryViewController.swift]: Utils isn't quite the right place. Yeah, create a Constants.swift.

**Pull request merged**